source/containers.tex

```diff
@@ -16086,6 +16093,1215 @@
+whose key is equivalent to \tcode{k}.
+\end{itemdescr}
+
+\indexlibrarymember{insert}{flatset}%
+\begin{itemdecl}
+template<class InputIterator>
+  void insert(InputIterator first, InputIterator last);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Adds elements to \exposid{c} as if by:
+\begin{codeblock}
+c.insert(c.end(), first, last);
+\end{codeblock}
+Then,
+sorts the range of newly inserted elements with respect to \tcode{compare};
+merges the resulting sorted range and
+the sorted range of pre-existing elements into a single sorted range; and
+finally erases all but the first element
+from each group of consecutive equivalent elements.
+
+\pnum
+\complexity
+$N$ + $M \log M$, where $N$ is \tcode{size()} before the operation and
+$M$ is \tcode{distance(first, last)}.
+
+\pnum
+\remarks
+Since this operation performs an in-place merge, it may allocate memory.
+\end{itemdescr}
+
+\indexlibrarymember{insert}{flat_set}%
+\begin{itemdecl}
+template<class InputIterator>
+  void insert(sorted_unique_t, InputIterator first, InputIterator last);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{insert(first, last)}.
+
+\pnum
+\complexity
+Linear.
+\end{itemdescr}
+
+\indexlibrarymember{insert_range}{flat_set}%
+\begin{itemdecl}
+template<@\exposconcept{container-compatible-range}@<value_type> R>
+  void insert_range(R&& rg);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Adds elements to \exposid{c} as if by:
+\begin{codeblock}
+for (const auto& e : range) {
+  c.insert(c.end(), e);
+}
+\end{codeblock}
+Then,
+sorts the range of newly inserted elements with respect to \tcode{compare};
+merges the resulting sorted range and
+the sorted range of pre-existing elements into a single sorted range; and
+finally erases all but the first element
+from each group of consecutive equivalent elements.
+
+\pnum
+\complexity
+$N$ + $M \log M$, where $N$ is \tcode{size()} before the operation and $M$
+is \tcode{distance(first, last)}.
+
+\pnum
+\remarks
+Since this operation performs an in-place merge, it may allocate memory.
+\end{itemdescr}
+
+\indexlibrarymember{swap}{flat_set}%
+\begin{itemdecl}
+void swap(flat_set& y) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+ranges::swap(compare, y.compare);
+ranges::swap(c, y.c);
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{extract}{flatset}%
+\begin{itemdecl}
+container_type extract() &&;
+\end{itemdecl}
+
+\pnum
+\ensures
+\tcode{*this} is emptied, even if the function exits via an exception.
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{std::move(c)}.
+\end{itemdescr}
+
+\indexlibrarymember{replace}{flat_set}%
+\begin{itemdecl}
+void replace(container_type&& cont);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+The elements of \tcode{cont} are sorted with respect to \tcode{compare} and
+contain no equal elements.
+
+\pnum
+\effects
+Equivalent to: \tcode{c = std::move(cont);}
```

Comment:
```suggestion
Equivalent to: \tcode{c = std::move(cont)}.
```

Reply:
with a colon, a statement follows

Reply:
I think the status quo is right. The editing style says, "plain `expression` should appear only if it is of type void". The assignment expression does not have type void, so leaving it as a statement is fine.

I'm changing a separate "Equivalent to: \tcode{insert(...)}." to _remove_ the colon because the `insert` expression is indeed void.